### DISCUSSION TITLE
HDD state (active/sleep)

### DISCUSSION THREAD

**Discussion by Lipown** [dougiteixeira/proxmoxve#267] [Feature request] [answered]
Would it be possible to monitor if HDD is sleeping or is active please? That should be readable from syslog / smartctl

**Comment by dougiteixeira** [comment, marked as answer]
I don't think this is a feature that will be added, we already realized that searching for disk information causes the disks to wake up, so this feature would prevent the disks from sleeping.

**Reply by Lipown** [reply]
How I solved it i trigger this script on my host to send the state to home assistant without interrupting the sleep. Maybe it will help if this feature will be on the list for future :)

```
STATE=$(/usr/sbin/smartctl -i -n standby /dev/sda | grep -e "Device is in STANDBY mode" -e "Power mode is:    ACTIVE or IDLE" 2>/dev/null)
if [ "$STATE" = "Device is in STANDBY mode, exit(2)" ]
then
# STANBY
VALUE="Standby"
else
if [ "$STATE" = "Power mode is:    ACTIVE or IDLE" ]
then
# ACTIVE or IDLE
VALUE="Active"
else
# ERROR
VALUE="Error"
fi
fi
curl -X POST -H "Authorization: Bearer whatever"        -H "Content-Type: application/json"        -d '{"state": "'$VALUE'", "attributes": {"friendly_name": "NAS_2"}}'        https://some.domain.with.ha/api/states/sensor.nas2
```
